code/src/sixsq/nuvla/server/resources/spec/module_application_helm.cljc

```diff
@@ -0,0 +1,73 @@
+(ns sixsq.nuvla.server.resources.spec.module-application-helm
+  (:require
+    [clojure.spec.alpha :as s]
+    [sixsq.nuvla.server.resources.spec.common :as common]
+    [sixsq.nuvla.server.resources.spec.container :as container]
+    [sixsq.nuvla.server.resources.spec.core :as core]
+    [sixsq.nuvla.server.resources.spec.deployment :as deployment]
+    [sixsq.nuvla.server.resources.spec.module-component :as module-component]
+    [sixsq.nuvla.server.resources.spec.module-application :as module-app]
+    [sixsq.nuvla.server.util.spec :as su]
+    [spec-tools.core :as st]))
+
+
+(s/def ::helm-repo-url
+  (-> (st/spec ::core/url)
+      (assoc :name "helm-repo-url"
+             :json-schema/description "Helm repo URL (http, https, oci)")))
+
+
+(s/def ::helm-chart-name
+  (-> (st/spec ::core/nonblank-string)
+      (assoc :name "helm-chart-name"
+             :json-schema/description "Helm chart name")))
+
+
+(s/def ::helm-absolute-url
+  (-> (st/spec ::core/url)
+      (assoc :name "helm-absolute-url"
+             :json-schema/description "Helm chart absolute URL")))
+
+
+(s/def ::helm-chart-version
+  (-> (st/spec ::core/nonblank-string)
+      (assoc :name "helm-chart-version"
+             :json-schema/description "Helm chart version to deploy, latest (if empty)")))
+
+
+(s/def ::helm-repo-creds
```

Comment:
Is this a credential that allow to access the helm registry?

Reply:
Yes. I updated the docu string.